packages/renderer/src/lib/container/container-utils.ts

```diff
@@ -136,6 +138,19 @@ export class ContainerUtils {
     return containerInfo.engineName;
   }
 
+  getImageHref(containerInfo: ContainerInfo): string {
+    if (containerInfo.ImageBase64RepoTag) {
+      const imageTag = Buffer.from(containerInfo.ImageBase64RepoTag, 'base64').toString();
+      if (imageTag.startsWith('sha256:')) {
+        return `/images/${containerInfo.ImageID}/${containerInfo.engineId}`;
+      } else {
+        return `/images/${containerInfo.ImageID}/${containerInfo.engineId}/${containerInfo.ImageBase64RepoTag}/summary`;
+      }
+    } else {
+      return `/images/${containerInfo.ImageID}/${containerInfo.engineId}`;
+    }
+  }
+
```

Comment:
it seems we are not testing the case without sha256 ?

Reply:
@benoitf I'm not sure I understand what you mean. Do you mean if there is no tag, but the `imageId` doesn't start with `sha256` -> add `containerInfo.ImageBase64RepoTag === containerInfo.ImageId` to the `if` statement in line 144?

Reply:
sorry I wasn't clear enough
I mean to have tests in the spec.ts file covering all the branches of your new method

Reply:
@benoitf Line 323 (in the `speac.ts` file) tests the url with an image tag, and the test in line 337 checks the url with an image id instead of a tag, but it looks like I'll have to add another one for the case of `if (containerInfo.ImageBase64RepoTag)`